x/multi-staking/keeper/invariants.go

```diff
@@ -42,7 +42,7 @@ func ModuleAccountInvariants(k Keeper) sdk.Invariant {
 		moduleAccount := authtypes.NewModuleAddress(types.ModuleName)
 		escrowBalances := k.bankKeeper.GetAllBalances(ctx, moduleAccount)
 
-		broken := !escrowBalances.IsEqual(totalLockCoinAmount)
```

Comment:
(https://github.com/cosmos/cosmos-sdk/pull/14739) Deprecate the method `Coin.IsEqual` in favour of  `Coin.Equal`.